File: src/WebAPI/WebAPI/Models/Communication/CalculateRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace WebAPI.Models.Communication
{
    public class CalculateRequest
    {
        public bool Initialized { get; set; }
        public string RepresentType { get; set; }
        public string AlgorithmType { get; set; }
        public Solution UnsolvedSolution { get; set; }
    }
}

File: src/algorithm-service/CustomSolver/src/main/java/wzy/jsp/simplest/demo/domain/communication/amqp/CalculateRequest.java
package wzy.jsp.simplest.demo.domain.communication.amqp;

import wzy.jsp.simplest.demo.domain.communication.Solution;

public class CalculateRequest {
    public Boolean Initialized=false;
    public String RepresentType;
    public String AlgorithmType;
    public Solution UnsolvedSolution;
}
